### PR TITLE
Update lateny report output message

### PR DESCRIPTION
**Pull request by hwware**
1. Update "Redis" to "Valkey"
2. Remove Dave dialog, and  update to normal output message

I think this should be not break change.

### CODE REVIEW

**Review by enjoy-binbin — APPROVED** [review]
yean, it look like a minor changes

File: src/latency.c [hunk 1]
```diff
@@ -372,39 +372,39 @@ sds createLatencyReport(void) {
...
+            report = sdscatprintf(report,"- There are latency issues with potentially slow commands you are using. Try to enable the Slow Log Valkey feature using the command 'CONFIG SET slowlog-log-slower-than %llu'. If the Slow log is disabled Valkey is not able to log slow commands execution for you.\n", (unsigned long long)server.latency_monitor_threshold*1000);
         }
 
         if (advise_slowlog_tuning) {
             report = sdscatprintf(report,"- Your current Slow Log configuration only logs events that are slower than your configured latency monitor threshold. Please use 'CONFIG SET slowlog-log-slower-than %llu'.\n", (unsigned long long)server.latency_monitor_threshold*1000);
         }
 
         if (advise_slowlog_inspect) {
-            report = sdscat(report,"- Check your Slow Log to understand what are the commands you are running which are too slow to execute. Please check https://redis.io/commands/slowlog for more information.\n");
+            report = sdscat(report,"- Check your Slow Log to understand what are the commands you are running which are too slow to execute. Please check https://valkey.io/commands/slowlog/ for more information.\n");
```
Comment (zuiderkwast) [inline src/latency.c:397]: The command pages are not dirs so I think a trailing slash makes it a broken link.

```suggestion
            report = sdscat(report,"- Check your Slow Log to understand what are the commands you are running which are too slow to execute. Please check https://valkey.io/commands/slowlog for more information.\n");
```

**Review by zuiderkwast — COMMENTED** [review]
I'm a little sad to remove the Dave dialog. It's a reference to the film 2001 A Space Odyssey. Have you watched it? It's a brilliant film from early time, 1969 IIRC. The computer talks to the crewman Dave on a space ship.

The film is producted in 1969 https://en.wikipedia.org/wiki/2001:_A_Space_Odyssey

**Comment by enjoy-binbin** [comment]
> I'm a little sad to remove the Dave dialog. It's a reference to the film 2001 A Space Odyssey. Have you watched it? It's a brilliant film from early time, 1969 IIRC. The computer talks to the crewman Dave on a space ship.

wow, i haven't watched it, don't know the history. I did a search and it looks like a very exciting film. Can't imagine a movie this old, i am going to add it to my list and then try to find time to take a look.

If that's the case, I think maybe we can keep it? It seems that the modification is not particularly important, but it seems to be very memorable.

**Comment by zuiderkwast** [comment]
https://youtu.be/Mme2Aya_6Bc

https://youtu.be/twTESNFc1ww

**Comment by zuiderkwast** [comment]
@madolson Let's keep the Dave dialog?

**Comment by hwware** [comment]
> I'm a little sad to remove the Dave dialog. It's a reference to the film 2001 A Space Odyssey. Have you watched it? It's a brilliant film from early time, 1969 IIRC. The computer talks to the crewman Dave on a space ship.

Sorry, I never see this movie. Now i know where Dave comes from, I am just curious why Dave here. 
@madolson DO you think we should keep Dave here?

File: src/latency.c [hunk 2]
```diff
@@ -372,39 +372,39 @@ sds createLatencyReport(void) {
     }
 
     if (eventnum == 0 && advices == 0) {
-        report = sdscat(report,"Dave, no latency spike was observed during the lifetime of this Redis instance, not in the slightest bit. I honestly think you ought to sit down calmly, take a stress pill, and think things over.\n");
+        report = sdscat(report,"No latency spike was observed during the lifetime of this Valkey instance, not in the slightest bit.\n");
     } else if (eventnum > 0 && advices == 0) {
-        report = sdscat(report,"\nWhile there are latency events logged, I'm not able to suggest any easy fix. Please use the Redis community to get some help, providing this report in your help request.\n");
+        report = sdscat(report,"\nThere are latency events logged, they are not easy fix. Please get some help from Valkey community, providing this report in your help request.\n");
```
Comment (madolson) [inline src/latency.c:377]:
```suggestion
        report = sdscat(report,"\nThere are latency events logged, they are not easy to fix. Please get some help from Valkey community, providing this report in your help request.\n");
```

**Comment by madolson** [comment]
I have had more than one conversation with someone asking who "dave" was, and then I had to explain to them how it's a reference to 2001 a space odyssey, and then they told me they have never heard of the movie. Some of them are also weird, like "I honestly think you ought to sit down calmly, take a stress pill, and think things over". I think someone might take that the wrong way.

"I'm sorry, Dave, I can't do that." Is the iconic line, it's also on an error path since latency is disabled. So I guess my vote is let's just keep that one and delete all the other references.

File: src/latency.c [hunk 3]
```diff
@@ -372,39 +372,39 @@ sds createLatencyReport(void) {
...
     } else {
         /* Add all the suggestions accumulated so far. */
 
         /* Better VM. */
-        report = sdscat(report,"\nI have a few advices for you:\n\n");
+        report = sdscat(report,"\nHere are a few advices for you:\n\n");
```
Comment (madolson) [inline src/latency.c:382]:
```suggestion
        report = sdscat(report,"\nHere is some advice for you:\n\n");
```

File: src/latency.c [hunk 4]
```diff
@@ -437,19 +437,19 @@ sds createLatencyReport(void) {
         }
 
         if (advise_hz && server.hz < 100) {
-            report = sdscat(report,"- In order to make the Redis keys expiring process more incremental, try to set the 'hz' configuration parameter to 100 using 'CONFIG SET hz 100'.\n");
+            report = sdscat(report,"- In order to make the Valkey keys expiring process more incremental, try to set the 'hz' configuration parameter to 100 using 'CONFIG SET hz 100'.\n");
         }
 
         if (advise_large_objects) {
             report = sdscat(report,"- Deleting, expiring or evicting (because of maxmemory policy) large objects is a blocking operation. If you have very large objects that are often deleted, expired, or evicted, try to fragment those objects into multiple smaller objects.\n");
```
Comment (madolson) [inline src/latency.c:444]: Maybe pin for a second issue, we should consider updating this to reference to include lazy deletion.

File: src/latency.c [hunk 5]
```diff
@@ -437,19 +437,19 @@ sds createLatencyReport(void) {
...
         }
 
         if (advise_mass_eviction) {
-            report = sdscat(report,"- Sudden changes to the 'maxmemory' setting via 'CONFIG SET', or allocation of large objects via sets or sorted sets intersections, STORE option of SORT, Redis Cluster large keys migrations (RESTORE command), may create sudden memory pressure forcing the server to block trying to evict keys. \n");
+            report = sdscat(report,"- Sudden changes to the 'maxmemory' setting via 'CONFIG SET', or allocation of large objects via sets or sorted sets intersections, STORE option of SORT, Valkey Cluster large keys migrations (RESTORE command), may create sudden memory pressure forcing the server to block trying to evict keys. \n");
```
Comment (madolson) [inline src/latency.c:448]: Maybe pin for a second issue. We incrementally do mass evictions now, so I'm not sure this is much of an issue anymore.

**Review by madolson — COMMENTED** [review]
Minor grammar fixes, and some of these events no longer seem relevant.

**Comment by zuiderkwast** [comment]
I'm fine with removing most of these, if the rest of you want that.

> "I honestly think you ought to sit down calmly, take a stress pill, and think things over"

This one is in the 2nd of the two youtube videos I posted above.

**Comment by madolson** [comment]
> This one is in the 2nd of the two youtube videos I posted above.

I had seen the movie and had no recollection of that phrase 😓. If other people do think it's popular, I would be fine keeping it, I just do think it's much less more recognizable than the "I can't do that" line.

**Comment by zuiderkwast** [comment]
Can we agree on keeping only one "i can't do that dave" and delete the others? (👍 = yes)

**Comment by zuiderkwast** [comment]
Replaced by #644

**Pull request closed without merging**